### ISSUE TITLE
Git [...] menu not show translated text

### ISSUE THREAD

**Issue by edp1096** [Microsoft/vscode-loc#38]
vscode version : 1.26.1
![default](https://user-images.githubusercontent.com/15924482/44306449-fd67ea80-a3c9-11e8-8906-0bf39eba5523.png)

**Comment by danyeh** [comment]
@edp1096 , thank you for your feedback! Could you please update to Korean language pack 1.26.3 if you haven't? I could not repro the issue on VS Code 1.26.1 + Korean Language Pack 1.26.3.
![image](https://user-images.githubusercontent.com/19785531/44354503-dad3e100-a45e-11e8-9a5f-7c47eba39f65.png)

**Comment by edp1096** [comment]
Hello @danyeh , I checked the version is 1.26.3 and found problem caused from userdata/clp folder.

Could you please check attached files?
[clp_normal_clean_install.zip](https://github.com/Microsoft/vscode-loc/files/2303425/clp_normal_clean_install.zip)
[clp_abnormal_my_set.zip](https://github.com/Microsoft/vscode-loc/files/2303426/clp_abnormal_my_set.zip)

**Comment by danyeh** [comment]
Thank you @edp1096 for the additional info! 
@dbaeumer @sandy081 @ramya-rao-a , please take a look and re-assign.

**Comment by dbaeumer** [comment]
I looked at the zips and there is nothing abnormal even with the clp_abnormal_my_set.zip. The git files are identical and the tcf.json files only differ in a small path segment (e.g. PC9 <-> swshi)

Did you copy that folder ?

**Comment by edp1096** [comment]
Yes.
Because maintain same settings, I'm using zipped vscode with --user-data-dir option at my PCs.

**Comment by dbaeumer** [comment]
Actually copying the user data dir from one PC to another is not supported right now.

**Comment by dbaeumer** [comment]
@edp1096 yes, that is correct. We have issues there as well. A corresponding item exists in the vscode repository
